src/component/navigation.ts

```diff
@@ -36,4 +36,12 @@ export class Navigation {
   zoom(type: ZoomType): void {
     type == 'in' ? this.graph.zoomIn() : this.graph.zoomOut();
   }
+
+  enable(): void {
```

Comment:
**thought:** It looks like it is the right place to put the methods in the API and putting the actual implementation in the `BpmnGraph` object seems also accurate and consistent with the rest of the implementation.
We may have a single method which takes the "enabling status" (boolean) as parameter. I currently don't have strong opinion about this. For now, having 2 methods make sense to me.